src/pages/CollectionsPage.tsx

```diff
@@ -1,16 +1,34 @@
-import { Fragment } from "react"
-import HeaderBar from "../components/HeaderBar/HeaderBar"
+import { Fragment, useEffect, useState } from "react"
 import { Container, Row, Col } from 'react-bootstrap';
+import { Collection } from "@imtbl/core-sdk";
+import HeaderBar from "../components/HeaderBar/HeaderBar"
+import GetYourCollections from "../api/collections-api";
 
 const CollectionsPage = () => {
+    const [result, setResult] = useState<[] | Array<Collection>>([])
+
+    // inspired by this Stackflow answer:https://stackoverflow.com/questions/53332321/react-hook-warnings-for-async-function-in-useeffect-useeffect-function-must-ret
+    useEffect(() => {
+        async function fetchCollection() {
+            let response = await GetYourCollections()
+            setResult(response.result)
+        }
+        fetchCollection()
+    }, [])
+
     return (
         <Fragment>
             <HeaderBar />
             <Container>
                 <Row>
-                    <Col>
-                        {/* This is where the cards will be displayed */}
-                    </Col>
+                    {result.length > 0 && result.map((item: Collection) => {
```

Comment:
If `result.length` is NOT 0, then `result.length` will evaluate to TRUE.

We can shorten this to
```
{result.length && result.map((item: Collection) => {
```

https://www.freecodecamp.org/news/falsy-values-in-javascript/